src/applications/vaos/utils/appointment.js

```diff
@@ -47,7 +47,7 @@ export const PAST_APPOINTMENTS_HIDE_STATUS_SET = new Set([
  */
 
 export function getRealFacilityId(facilityId) {
-  if (!environment.isProduction() && facilityId) {
+  if ((!environment.isProduction() || window.Cypress) && facilityId) {
```

Comment:
Added 'window.Cypress' since the environment is surprisingly production in the CI build. I thought it would be staging.